packages/react-core/src/components/DualListSelector/DualListSelector.tsx

```diff
@@ -0,0 +1,394 @@
+        if (prevState.chosenOptionsSelected.indexOf(index) !== -1) {
+          itemsToRemove.push(value);
+        } else {
+          newChosen.push(value);
+        }
+      });
+
+      const newAvailable = [...prevState.availableOptions, ...itemsToRemove];
+      if (this.props.removeSelected) {
+        this.props.removeSelected(newAvailable, newChosen);
+      }
+      if (this.props.onListChange) {
+        this.props.onListChange(newAvailable, newChosen);
+      }
+      return {
+        chosenOptionsSelected: [],
+        availableOptionsSelected: [],
+        chosenOptions: newChosen,
+        availableOptions: newAvailable
+      };
+    });
+  };
+
+  onOptionSelect = (e: React.MouseEvent | React.ChangeEvent, index: number, isChosen: boolean) => {
+    this.setState(prevState => {
+      const originalArray = isChosen ? prevState.chosenOptionsSelected : prevState.availableOptionsSelected;
+
+      let updatedArray = null;
+      if (originalArray.indexOf(index) !== -1) {
+        updatedArray = originalArray.filter(value => value !== index);
+      } else {
+        updatedArray = [...originalArray, index];
+      }
+
+      return {
+        chosenOptionsSelected: isChosen ? updatedArray : prevState.chosenOptionsSelected,
+        availableOptionsSelected: isChosen ? prevState.availableOptionsSelected : updatedArray
+      };
+    });
+
+    const onOptionSelect = this.props.onOptionSelect;
+    if (onOptionSelect) {
+      onOptionSelect(e);
+    }
+  };
+
+  handleKeys = (event: KeyboardEvent) => {
+    const key = event.key;
+    let moveFocus = false;
+    let currentIndex = -1;
+    const controls = Array.from(this.controlsEl.current.getElementsByClassName('pf-c-button'));
+    if (['ArrowUp', 'ArrowDown', 'ArrowLeft', 'ArrowRight'].includes(event.key)) {
+      if (document.activeElement === this.controlsEl.current) {
+        currentIndex = 0;
+        while (
+          currentIndex < controls.length &&
+          currentIndex >= 0 &&
+          controls[currentIndex].classList.contains('pf-m-disabled')
+        ) {
+          currentIndex = currentIndex + 1;
+        }
+        moveFocus = true;
+      } else {
+        controls.forEach((control, index) => {
+          if (document.activeElement === control) {
+            const increment = key === 'ArrowUp' || key === 'ArrowLeft' ? -1 : 1;
+            currentIndex = index + increment;
+            while (
+              currentIndex < controls.length &&
+              currentIndex >= 0 &&
+              controls[currentIndex].classList.contains('pf-m-disabled')
+            ) {
+              currentIndex = currentIndex + increment;
+            }
+            moveFocus = true;
+          }
+        });
+      }
+      event.preventDefault();
+    }
+    if (moveFocus && controls[currentIndex]) {
+      (controls[currentIndex] as HTMLElement).focus();
+    }
+  };
+
+  componentDidMount() {
+    window.addEventListener('keydown', this.handleKeys);
```

Comment:
This is killing all arrow key functionality for me as it calls `event.preventDefault()` on line 259 above for any arrow key - even outside of the component this is disabling the arrow keys, maybe @jessiehuff knows a better way to attach this listener

Reply:
oh you're right, the preventDefault is in the wrong place :)